sagemaker-experiments/associate-hyper-parameter-tuning-job/associate-hyper-parameter-tuning-job.ipynb

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
<p>Hyperparameter is one word despite what spellcheck says.</p><p>Prerequisite spelling</p><p>Use consistent verbs in the steps. Retrieve, create, associate (-s)</p><p> </p><p>What kernel should this use?</p><p>Pin the sagemaker version required (if there are any SDK v2 vs 1 issues in the calls made)</p><p> </p><p>Add an outro: cleanup if needed, summarize what you just did, discuss results, and call to action (you could move contact down there).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
updated